package.json

```diff
@@ -1,10 +1,20 @@
 {
-  "name": "organizer",
+  "name": "@organizer/root",
```

Comment:
why `@`?

Reply:
It does not really matter how this is called because the main root folder only is liable for delivering the better way of managing many packages from one single place and this will never be published anywhere. The name has been changed because we need to stick to some rules, our organization is called "organizer" and because of the convention that libraries start with the "@" symbol our library is called @organizer. The postfix "/root" only denotes that this is where all code and data is stored.